eve/main.yml

```diff
@@ -663,6 +663,17 @@ models:
         create-env --name "$ENVIRONMENT_NAME"
       workdir: build/eve/workers/openstack-terraform/terraform/
       haltOnFailure: true
+  - ShellCommand: &untaint_bootstrap
+      doStepIf: "%(prop:install_solution:-false)s"
+      name: Untaint Bootstrap before deploying a Solution
+      env:
+        NODE_NAME: "bootstrap"
+      command: >
+        ssh -F ssh_config bootstrap
+        sudo kubectl --kubeconfig /etc/kubernetes/admin.conf
+        patch node "$NODE_NAME" --patch '{"spec": {"taints": []}}'
```

Comment:
:facepalm: Gah this fails because the quotes are removed by the first shell, before the command is sent through SSH...